airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowTest.java

```diff
@@ -1449,53 +1396,6 @@ public void failedResetContinueAsReset() throws InterruptedException {
 
     }
 
-    @RepeatedTest(10)
-    @Timeout(value = 2,
-             unit = TimeUnit.SECONDS)
-    @DisplayName("Test that we are getting stuck if the report of a failure happen")
-    void testGetStuckAfterRun() throws InterruptedException {
```

Comment:
Similar motivation around removing this test - the behavior is already being tested by the `testWorkflowRestartedAfterFailedActivity()` test above